Virology dispensers

DreadedReborn opened the pull request:
:cl: DreadedReborn
add: New virology dispensers. Synaptizine, Unstable Mutagen, Radium and Liquid Phoron
imageadd: Virology dispensers come with some sprites, provided by NijiShadow.
/:cl:

Commit:
Merge remote-tracking branch 'refs/remotes/origin/dev'

Commit:
Virology changes

yes

CakeQ commented:
It would be best to move the new dispensers into a file somewhere in modules_apollo instead of injecting new code into actual TGcode so that we don't have to deal with merge issues later on when updating. Something like modules_apollo/misc/chem_dispenser.dm or something.

CakeQ commented:
test

CakeQ commented:
test

CakeQ commented:
test

Commit:
Virology placement fix

Moves chem dispensers to modules_apollo so they aren't overriden by /tg/
pulls.
